src/main/resources/application-hsqldb.properties

```diff
@@ -1,5 +1,6 @@
 # HSQLDB config start
 #----------------------------------------------------------------
+database=hsqldb
```

Comment:
some of your testing properties again? I am not sure this property even does anything, tbh

Reply:
Ah, I see, good point. It's not contained in the `h2` profile though.
Also in any case it's better not to add something, if it's not needed or if you don't exactly know what it does.